ocp_resources/cdi_config.py

```diff
@@ -54,3 +54,9 @@ def wait_until_upload_url_changed(self, uploadproxy_url, timeout=TIMEOUT_4MINUTE
                 current_url = status.uploadProxyURL
                 if current_url == uploadproxy_url:
                     return
+
+    @classmethod
+    def is_garbage_collector_enabled(cls):
+        """Check if garbage collector enabled on config CDIconfig resource"""
+        dv_ttl_seconds = cls(name="config").instance.spec.get("dataVolumeTTLSeconds")
```

Comment:
`instance.spec.get("dataVolumeTTLSeconds")`
should be 
`instance.spec.dataVolumeTTLSeconds`

Reply:
this is not mandatory field, if dataVolumeTTLSeconds does not exists is same as dataVolumeTTLSeconds: 0

Reply:
resource instance is a dict like object that will return None if not exists, you use the instance object then you should do:
`instance.spec.dataVolumeTTLSeconds`

Reply:
ack